src/persist-client/src/internal/state_versions.rs

```diff
@@ -973,7 +974,22 @@ impl<T: Timestamp + Lattice + Codec64> ReferencedBlobValidator<T> {
                 self.full_rollups.insert(x.clone());
             }
         });
-        assert_eq!(self.inc_batches, self.full_batches);
+
+        if let Some(first_incr) = self.inc_batches.first() {
```

Comment:
i'm still waking up, so this could be wrong, but can we assert that the set of parts is the same?

Reply:
yep! good call

Reply:
done!